maintainers/maintainer-list.nix

```diff
@@ -5712,4 +5712,9 @@
     github = "zzamboni";
     name = "Diego Zamboni";
   };
+  turbomack = {
+    email = "marek.faj@gmail.com";
+    github = "turboMaCk";
+    name = "Marek Fajkus";
+  };
```

Comment:
not sure if this is appropriate given I'm not a commiter - let me know if this should be excluded.

Reply:
You can be a maintainer without being a committer :)